governance/api-reviews.md

```diff
@@ -200,3 +200,56 @@ When adding new ops, look for:
    it CPU, GPU, XLA, TPU, etc) and prefer to have at least a plan for writing
    device-agnostic code
  - should the python layer for this operation support raggedtensor/sparsetensor?
+
+## Experimental APIs
+
+Experimental APIs are APIs which have the word 'experimental' somewhere in their
+name; for example `tf.experimental.foo`, or `tf.foo.experimental.Bar`, or
+`tf.foo(experimental_bar=True)` or `tf.Foo().experimental_bar()`.
```

Comment:
Do we have any tf.foo.ExperimentalBar()s?

Reply:
I don't think we should have tf.foo.ExperimentalBar; it should be tf.foo.experimental.Bar.

Reply:
edited the text to reflect this